shell/app/atom_content_client.cc

```diff
@@ -32,9 +32,10 @@
 #endif  // defined(WIDEVINE_CDM_AVAILABLE)
 
 #if BUILDFLAG(ENABLE_PDF_VIEWER)
-#include "pdf/pdf.h"
-#include "shell/common/atom_constants.h"
-#endif  // BUILDFLAG(ENABLE_PDF_VIEWER)
+#include "pdf/pdf.h"                      // nogncheck
```

Comment:
why `//nogncheck` ?

Reply:
beacuse gn check doesn't understand `#if`

Reply:
It should iirc if you have these as `public_deps` , I remember doing it for other guarded sources we have in `chromium_src/BUILD.gn`

Reply:
we already have the deps, they're just conditional in the gn file based on the `enable_pdf_viewer` build flag. i suppose i could contort to do it by putting this stuff in a separate target, but i don't think it's worth it.